components/layout/OpenAuthoringSiteForm.tsx

```diff
@@ -16,9 +18,12 @@ const OpenAuthoringSiteForm = ({ form, editMode, path, children }: Props) => {
   // Persist pending changes to localStorage
   useLocalStorageCache(path, form, editMode)
 
+  useUpdateAuthOnFormFail(form)
+
   return (
     <>
       <FormAlerts form={form} />
+      <AutoAuthModal />
```

Comment:
What do you think of moving `useUpdateAuthOnFormFail(form)` inside `AutoAuthModal`?

```
<AutoAuthModal form={form} />
```

Reply:
IMO they are pretty independent. 
We currently keep a semi-recent auth-state in the root OpenAuthoring component. This state determines if we need to re-authenticate the user / create a new fork when they click edit (which pops the window).

useUpdateAuthOnFormFail triggers the root to re-perform that check, so that when the user clicks "continue", if required, we can open the auth window synchronously (otherwise the window gets popup-blocked).

AutoAuthModal grabs a queryParm and shows a modal kicking off the auth-flow on intiialLoad. If we removed this, we'd still want "useUpdateAuthOnFormFail" to do it's thing when a runtime error shows up.

Reply:
I think I confused things by having `errorModalProps` as a variable name in AutoAuthModal (copy-paste mistake)